openqaoa/backends/simulators/qaoa_vectorized.py

```diff
@@ -20,7 +20,7 @@
 import numpy as np
 from copy import copy
 from scipy.sparse import csc_matrix, kron, diags
-from scipy.linalg import expm
+from scipy.sparse.linalg import expm
```

Comment:
wow, this was the only line that was deprecated? Thanks!

Reply:
Seems so! The vectorised backend is the only place we manipulate sparse matrices with scipy.linalg, so this was the only place the tests failed.